profile/README.md

```diff
@@ -0,0 +1,138 @@
+<div align="center">
+    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/E-constellatio.svg" alt="Logo" height="150px">
+    <h1>
+        Welcome in E-constellatio Github organization
+    <img src="https://media.giphy.com/media/hvRJCLFzcasrR4ia7z/giphy.gif" alt="Hi hand" width="30px"/>
+    </h1>
+</div>
+
+This organization contain repositories of E-constellatio website. Repositories of other products can be found
+in [Libra foundation organization](https://github.com/Libra-foundation)
```

Comment:
I do not much like how the phrasing groups Libra and correlatio, those two entities serve very different purposes.